backend/routes/calendar.routes.js

```diff
@@ -11,7 +11,8 @@ getAuthenticatedClient().then(auth => {
 
 router.post('/', async (req, res) => {
```

Comment:
We can remove async because we don't use await.

Reply:
Fixed.